lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -19,17 +20,42 @@ function canFindDenominatorInNumerator(node) {
   if (Node.Type.isParenthesis(denominator)) {
     denominator = denominator.content;
   }
+  if (!(numerator.op === '+' || numerator.op === '-' ||
+        denominator.op === '+' || numerator.op === '-')) {
+    return false;
+  }
+  if (denominator.op !== '+') {
+    return false;
+  }
+
+  let numeratorFirstTerm;
+  if (numerator.op === '+') {
+    numeratorFirstTerm = new Node.PolynomialTerm(numerator.args[0]);
```

Comment:
I'm concerned that we can just assume it'll be a polynomial term

(what if the second arg is a polynomial term? what if there are more than 2 args?)

Sorry this got more complicated haha. If you want, we can chat on gitter or something and figure out how to attack this